docs/website/docs/dlt-ecosystem/destinations/snowflake.md

```diff
@@ -136,7 +136,12 @@ If you set the [`replace` strategy](../../general-usage/full-loading.md) to `sta
 recreated with a [clone command](https://docs.snowflake.com/en/sql-reference/sql/create-clone) from the staging tables.
 
 ## Data loading
-The data is loaded using an internal Snowflake stage. We use the `PUT` command and per-table built-in stages by default. Stage files are immediately removed (if not specified otherwise).
+The data is loaded using an internal Snowflake stage. We use the `PUT` command and per-table built-in stages by default. Stage files are kept by default, unless specified otherwise via the `keep_staged_files` parameter:
```

Comment:
This is true only for Snowlake named or external stage (s3/azure/gcp). In this case, dlt will keep the latest files by default unless it says otherwise via `keep_staged_files` parameter. If people does not provide any stage to dlt, it uses table stage, and I don't know what happened to these files, but I don't think you can access them at all

Reply:
ahh ok so I guess I should edit this to say the stage files are removed unless you explicitly define a stage?